sso/resources/org.jaggeryjs.modules.sso/src/main/java/org/jaggeryjs/modules/sso/common/util/Util.java

```diff
@@ -49,6 +50,7 @@
 import javax.xml.parsers.DocumentBuilderFactory;
 import java.io.*;
 import java.security.KeyStore;
+import java.security.NoSuchAlgorithmException;
 import java.util.Random;
 import java.util.zip.*;
```

Comment:
Understood. :) I just found that it's been fixed now. Please find the updated file here. https://github.com/wso2/jaggery-extensions/blob/master/sso/resources/org.jaggeryjs.modules.sso/src/main/java/org/jaggeryjs/modules/sso/common/util/Util.java